HST/finalize_document.py

```diff
@@ -0,0 +1,163 @@
+##########################################################################################
+# finalize_document/__init__.py
+#
+# - Create document directory.
+# - Move/copy proposal files over from pipeline directory.
+# - Create document xml.
+# - Create document collection csv & xml.
+##########################################################################################
+
+import os
+import pdslogger
+import shutil
+
+from hst_helper import (DOCUMENT_EXT,
+                        PROGRAM_INFO_FILE)
+from hst_helper.fs_utils import (create_col_dir_in_bundle,
+                                 get_deliverable_path,
+                                 get_formatted_proposal_id,
+                                 get_program_dir_path)
+from hst_helper.general_utils import (create_xml_label,
+                                      create_collection_label,
+                                      create_csv,
+                                      get_instrument_id_set,
+                                      get_mod_history_from_label)
+
+DOC_LABEL_TEMPLATE = 'DOCUMENT_LABEL.xml'
+COL_DOC_LABEL_TEMPLATE = 'DOCUMENT_COLLECTION_LABEL.xml'
+CSV_FILENAME = 'collection.csv'
+COL_DOC_LABEL = 'collection.xml'
+
+def label_hst_document_directory(proposal_id, data_dict, logger=None, testing=False):
+    """With a given proposal id, create document directory in the final bundle. Return a
+    tupel of the path of document collection label and the path of document label. These
+    are the actions performed:
+    1. Create document directory.
```

Comment:
I would find all these comments easier to read if there was a blank line between the first paragraph and the numbered list.